src/components/home/sprint-view-card.tsx

```diff
@@ -19,17 +23,17 @@ const SprintViewCard = () => {
           >
             {adminMode ? (
               <CardContent>
-              <Typography variant="h6" fontWeight={"bold"} style={{ marginTop: 6, marginBottom: 3 }}>
-              {strings.sprint.sprintview}
-              </Typography>
-              <Typography variant="body1">{strings.placeHolder.notYetImplemented}</Typography>
-              </CardContent>
-            ) : (
-                <CardContent>
                 <Typography variant="h6" fontWeight={"bold"} style={{ marginTop: 6, marginBottom: 3 }}>
```

Comment:
As a general rule, if a component has 3 or more props they should each be indented on a new line, this improves code readability.